go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/NVIDIA/holodeck
 
-go 1.21.5
+go 1.22.0
```

Comment:
Not that we don't want to do this, but what is the driver for this change? Do we depend on newer language features?

Reply:
```suggestion
go 1.22
```
See https://go.dev/doc/modules/gomod-ref#go

Reply:
Is not a required change, but a good to have one.